api/tests/src/task/notifications/test_sync_suppressed_emails.py

```diff
@@ -0,0 +1,88 @@
+from datetime import UTC, datetime, timedelta
+
+import pytest
+
+from src.adapters.aws.sesv2_adapter import MockSESV2Client, SuppressedDestination
+from src.db.models.user_models import LinkExternalUser, SuppressedEmail
+from src.task.notifications.sync_suppressed_emails import SyncSuppressedEmailsTask
+from tests.lib.db_testing import cascade_delete_from_db_table
+from tests.src.db.models import factories
+
+
+@pytest.fixture(autouse=True)
+def clear_data(db_session):
+    cascade_delete_from_db_table(db_session, SuppressedEmail)
+    cascade_delete_from_db_table(db_session, LinkExternalUser)
+
+
+@pytest.fixture
+def sync_suppressed_emails(db_session):
+    client = MockSESV2Client()
+    task = SyncSuppressedEmailsTask(db_session, client)
+    return task, client
+
+
+def test_sync_suppressed_emails_update(sync_suppressed_emails, db_session, enable_factory_create):
+    task, client = sync_suppressed_emails
```

Comment:
Split this into two fixtures and you won't need to unpack the tuple in each test. The `mock_client` passed to the `task` fixture and to the test are the exact same object.
```suggestion
@pytest.fixture
def mock_client():
    return MockSESV2Client()
    
@pytest.fixture
def task(db_session, mock_client):
    task = SyncSuppressedEmailsTask(db_session, client)
    return task

def test_sync_suppressed_emails_update(task, mock_client, db_session, enable_factory_create):
```